shared/idmapset.go

```diff
@@ -26,9 +26,9 @@ type idmapEntry struct {
 
 func (e *idmapEntry) ToLxcString() string {
 	if e.isuid {
-		return fmt.Sprintf("u %d %d %d\n", e.nsid, e.hostid, e.maprange)
+		return fmt.Sprintf("u %d %d %d", e.nsid, e.hostid, e.maprange)
 	}
-	return fmt.Sprintf("g %d %d %d\n", e.nsid, e.hostid, e.maprange)
+	return fmt.Sprintf("g %d %d %d", e.nsid, e.hostid, e.maprange)
```

Comment:
That change will affect the container's configuration, likely leading to an invalid configuration.